js/traffic_car.js

```diff
@@ -1,65 +0,0 @@
-  var trafficCarOps = {
-      windowHeight: null,
-      widthMax: null,
-      trafficCar1: function (min, max) {
-          var leftCar = '<div class="obCar1 obCar" id="leftRandomCar"></div>';
-          $("#road").append(leftCar);
-          $("#leftRandomCar").css({
-              //"left": (Math.floor(Math.random() * 10) + 40) + "%"
-              "left": (Math.floor(Math.random() * (max - min + 1) + min)) + "%"
-          });
-      },
-
-      trafficCar2: function (min, max) {
-          var rightCar = '<div class="obCar2 obCar" id="rightRandomCar"></div>';
-          $("#road").append(rightCar);
-          $("#rightRandomCar").css({
-              //"left": (Math.floor(Math.random() * 10) + 65) + "%"
-              "left": (Math.floor(Math.random() * (max - min + 1) + min)) + "%"
-          });
-      },
-      trafficCar3: function (min, max) {
-          var middleCar = '<div class="obCar3 obCar" id="midRandomCar"></div>';
-          $("#road").append(middleCar);
-          $("#midRandomCar").css({
-              //"left": Math.floor(Math.random() * 20) + "%"
-              "left": (Math.floor(Math.random() * (max - min + 1) + min)) + "%"
-          });
-      },
-      intrvlTrafficCar1: function () {
```

Comment:
intrvlTrafficCar1 and intrvlTrafficCar2 sharing same code...write a single function with parameters